tests/acceptance/steps/unsubscribe_topics.py

```diff
@@ -43,3 +45,23 @@ def step_impl4(context):
     with ZK(cluster_config) as zk:
         offsets = zk.get_group_offsets(context.group)
     assert context.topic not in offsets
+
+
+@when('we call the unsubscribe_topics command with kafka storage')
+def step_impl2_kafka(context):
+    call_unsubscribe_topics(context.group, 'kafka')
```

Comment:
since kafka is the default storage, to we should not pass storage option. This would be a more real scenario. What do you think?